test/SIMD.int16x8.asmjs/testShift.js

```diff
@@ -96,4 +167,11 @@ equalSimd([27136, 0, -2560, 14848, 30720, 8192, 4864, 1280], m.testShiftLeftScal
 equalSimd([106, 0, -10, 58, 120, 32, 19, 5], m.testShiftRightScalarGlobal(), SIMD.Int16x8, "testMaxGlobal");
 equalSimd([-12800, -6144, 5120, -16640, 0, -32256, 0, -17920], m.testShiftLeftScalarGlobalImport(), SIMD.Int16x8, "testMinGlobalImport");
 equalSimd([-50, -24, 20, -65, 0, -126, 0, -70], m.testShiftRightScalarGlobalImport(), SIMD.Int16x8, "testMaxGlobalImport");
+
+equalSimd([-30720, -18688, -26368, 26624, 19456, 4608, 17408, -6400], m.testShiftLeftScalarLocal1(), SIMD.Int16x8, "testMinLocal1");
+equalSimd([19, -14, 2, -13, -4, 11, 30, -1], m.testShiftRightScalarLocal1(), SIMD.Int16x8, "testMaxLocal1");
+equalSimd([27136, 0, -2560, 14848, 30720, 8192, 4864, 1280], m.testShiftLeftScalarGlobal1(), SIMD.Int16x8, "testMinGlobal1");
+equalSimd([106, 0, -10, 58, 120, 32, 19, 5], m.testShiftRightScalarGlobal(), SIMD.Int16x8, "testMaxGlobal1");
+equalSimd([-12800, -6144, 5120, -16640, 0, -32256, 0, -17920], m.testShiftLeftScalarGlobalImport1(), SIMD.Int16x8, "testMinGlobalImport1");
+equalSimd([-50, -24, 20, -65, 0, -126, 0, -70], m.testShiftRightScalarGlobalImport1(), SIMD.Int16x8, "testMaxGlobalImport1");
 print("PASS");
```

Comment:
equalSimd(..) from ` this.WScript.LoadScriptFile("..\\UnitTestFramework\\SimdJsHelpers.js");` will print a failure message if the values are incorrect. RL.exe expects only "PASS" on success. On failure, it will failure message + "PASS"".